src/services/balancer/subgraph/entities/pools/__mocks__/decoratedPool.ts

```diff
@@ -0,0 +1,83 @@
+// import { Pool } from '../../../types';
+
+const decoratedPool: any = {
```

Comment:
@timjrobinson any ideas why if I import the `Pool` type jest dies? It complains about the defined `enum` there... perhaps its not compatible with jest quite yet?

Reply:
nvm its working now.